### PR TITLE
Quagga routing suite

### PR DESCRIPTION
**Pull request by seanchannel**
Here's a formula for the current stable version of Quagga.

The long ``./configure`` enables all the daemons/protocols and especially the remote admin/CLI tool ``vtysh``, which is used to remotely administer these daemons and general routing on other routers (as well as maybe locally).

Thanks to mistym on IRC

### CODE REVIEW

File: Library/Formula/quagga.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  test do
+    # `test do` will create, run in and delete a temporary directory.
+    #
+    # This test will fail and we won't accept that! It's enough to just replace
+    # "false" with the main program this formula installs, but it'd be nice if you
+    # were more thorough. Run the test with `brew test quagga`. Options passed
+    # to `brew install` such as `--HEAD` also need to be provided to `brew test`.
+    #
+    # The installed folder is not in the path, so use the entire path to any
+    # executables being tested: `system "#{bin}/program", "do", "something"`.
```
Comment (bfontaine) [inline Library/Formula/quagga.rb]: Please remove this comment.

File: Library/Formula/quagga.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Quagga Routing Suite
```
Comment (bfontaine) [inline Library/Formula/quagga.rb]: You can remove this comment.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by seanchannel** [comment]
This took so long I deleted my fork/branch, plus Quagga has moved on to newer versions.

**Pull request closed without merging**